### PR TITLE
Set resources limit for e2e-sse pod

### PR DESCRIPTION
**Pull request by cydrain**

### CODE REVIEW

File: ci/pod/e2e-sse.yaml [hunk 1]
```diff
@@ -28,13 +28,13 @@ spec:
...
-    # resources:
-    #   requests:
-    #     memory: "16Gi"
-    #     cpu: "4"
-    #   limits:
-    #     memory: "16Gi"
-    #     cpu: "4"
+    resources:
+      requests:
+        memory: "3Gi"
```
Comment (liliu-z) [inline ci/pod/e2e-sse.yaml:33]: Per our previous talk, the host is 2C4G. Why we made it 16G before?

Comment (cydrain) [reply ci/pod/e2e-sse.yaml:33]: the host is 8c4g, Huangliang set no resource limit before

**Pull request merged**